tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -109,317 +49,306 @@ describe('Target aggregates', () => {
-      return [place, contact];
-    };
-    const docs = _.flattenDeep([
-      Array.from({ length: 5 }).map((e, i) => generatePlace(parentPlace, i, false)),
-      Array.from({ length: 5 }).map((e, i) => generatePlace(otherParentPlace, i, true)),
-    ]);
-    const generateTitle = (title) => ({ en: title });
-    const docTags = [
-      // current targets
-      moment().format('YYYY-MM'),
-      // next month targets, in case the reporting period switches mid-test
-      moment().date(10).add(1, 'month').format('YYYY-MM'),
-    ];
+  describe('User with one or more places assigned', () => {
+    const CURRENT_PERIOD = 'This month';
+    const NAMES_DH1 = ['Clarissa', 'Prometheus', 'Alabama', 'Jasmine', 'Danielle'];
+    const NAMES_DH2 = ['Viviana', 'Ximena', 'Esteban', 'Luis', 'Marta'];
+
+    const TARGET_VALUES_BY_CONTACT = helperFunctions.generateTargetValuesByContact([...NAMES_DH1, ...NAMES_DH2]);
+    
+    const districtHospital1 = placeFactory.place().build({ type: 'district_hospital', name: 'District Hospital 1' });
+    const districtHospital2 = placeFactory.place().build({ type: 'district_hospital', name: 'District Hospital 2' });
+
     const contactWithManyPlaces = personFactory.build({
-      parent: { _id: parentPlace._id, parent: { _id: parentPlace._id } },
+      parent: { _id: districtHospital1._id, parent: { _id: districtHospital1._id } },
     });
-    const userWithManyPlaces = {
+
+    const onlineUser = userFactory.build({ place: districtHospital1._id, roles: ['program_officer'] });
+    
+    const userWithManyPlaces = userSettingsFactory.build({
       _id: 'org.couchdb.user:offline_many_facilities',
-      language: 'en',
-      known: true,
-      type: 'user-settings',
+      id: 'org.couchdb.user:offline_many_facilities',
+      name: 'offline_many_facilities',
       roles: [ 'chw' ],
-      facility_id: [ parentPlace._id, otherParentPlace._id ],
-      contact_id: contactWithManyPlaces._id,
-      name: 'offline_many_facilities'
-    };
+      facility_id: [ districtHospital1._id, districtHospital2._id ],
+      contact_id: contactWithManyPlaces._id
+    });
     const userWithManyPlacesPass = uuid();
 
-    before(async () => {
-      const allDocs = [ ...docs, parentPlace, otherParentPlace, contactWithManyPlaces, userWithManyPlaces ];
-      await utils.saveDocs(allDocs);
-      await utils.createUsers([user]);
-      await utils.request({
-        path: `/_users/${userWithManyPlaces._id}`,
-        method: 'PUT',
-        body: { ...userWithManyPlaces, password: userWithManyPlacesPass, type: 'user' },
+    const contactDocs = [];
+    const targetDocs = [];
+    Array
+      .from({ length: 5 })
+      .forEach((e, i) => {
+        const district1Data =
+          helperFunctions.generateContactsAndTargets(districtHospital1, NAMES_DH1[i], TARGET_VALUES_BY_CONTACT);
+        const district2Data =
+          helperFunctions.generateContactsAndTargets(districtHospital2, NAMES_DH2[i], TARGET_VALUES_BY_CONTACT);
```

Comment:
Minor
```suggestion
        const district1Data = helperFunctions.generateContactsAndTargets(
          districtHospital1, NAMES_DH1[i], TARGET_VALUES_BY_CONTACT,
        );
        const district2Data = helperFunctions.generateContactsAndTargets(
          districtHospital2, NAMES_DH2[i], TARGET_VALUES_BY_CONTACT,
        );
```

Reply:
Thank you! I was not sure what the correct way to break that line was!